fuel/datasets/hdf5.py

```diff
@@ -106,16 +106,20 @@ class H5PYDataset(Dataset):
          array
       4. ``stop`` : stop index (exclusive) of the split in the source
          array
-      5. ``available`` : boolean, ``False`` is this split is not available
+      6. ``indices`` : h5py.Reference, reference to a dataset containing
+         subset indices for this split/source pair.
+      6. ``available`` : boolean, ``False`` is this split is not available
          for this source
-      6. ``comment`` : comment string
+      7. ``comment`` : comment string
 
     Parameters
     ----------
     file_or_path : :class:`h5py.File` or str
         HDF5 file handle, or path to the HDF5 file.
-    which_set : str
-        Which split to use.
+    which_set : str or tuple of str
```

Comment:
I made the change. I also changed the name from `which_set` to `which_sets` for two reasons:

1. It fits better with that this argument is now used for.
2. It decreases further the chance that someone will try to pass a string as `which_sets`.